src/scss/_functions.scss

```diff
@@ -48,19 +48,8 @@
 /// @example scss
 /// font-family: oFontsGetFontFamilyWithFallbacks(FinancierDisplayWeb);
 @function oFontsGetFontFamilyWithFallbacks($family) {
-	@if $family == 'Clarion' {
-		@warn "Clarion has been removed, use Georgia instead";
-		@return $o-fonts-serif;
-	}
-
-	@if $family == 'Benton' {
-		@warn "Benton is deprecated. It will be removed in the next major version (v3). Please use Metric instead.";
-	}
-	@if $family == 'Miller' {
-		@warn "Miller is deprecated. It will be removed in the next major version (v3). Please use Financier instead.";
-	}
-	@if $family == 'FinancierTextWeb' {
-		@warn "FinancierTextWeb is deprecated. It will be removed in the next major version (v3). Please use Georgia instead";
+	@if $family == 'Clarion' or $family == 'Benton' or $family == 'Miller' or $family == 'FinancierTextWeb' {
+		@warn "#{$family} has been removed, no font will be included.";
```

Comment:
Should this be a hard error? Since it's going to be in a major version. Would help people notice places they're being included if their builds suddenly break; it's easy to ignore warnings.